tests/table/__snapshots__/index.test.ts.snap

```diff
@@ -307,6 +307,210 @@ Array [
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "[https://external.com right]",
+            "text": "[https://external.com right]",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "[$ x]",
+            "text": "[$ x]",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "[N35.6812362,E139.7649361]",
+            "text": "[N35.6812362,E139.7649361]",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "#hashTag",
+            "text": "#hashTag",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "? helpfeel",
+            "text": "? helpfeel",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "$ commandLine",
+            "text": "$ commandLine",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "path": "progfay",
+            "pathType": "relative",
+            "raw": "[progfay.icon]",
+            "type": "icon",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "[https://image.com/image.png]",
+            "text": "[https://image.com/image.png]",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "content": "",
+            "href": "link",
+            "pathType": "relative",
+            "raw": "[link]",
+            "type": "link",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "plain",
+            "text": "plain",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "> quote",
+            "text": "> quote",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "[[progfay.icon]]",
+            "text": "[[progfay.icon]]",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "[[https://image.com/image.png]]",
+            "text": "[[https://image.com/image.png]]",
+            "type": "plain",
+          },
+        ],
+      ],
+      Array [
+        Array [
+          Object {
+            "raw": "[[strong]]",
+            "text": "[[strong]]",
+            "type": "plain",
+          },
+        ],
+      ],
+    ],
+    "fileName": "node in table cells",
+    "indent": 0,
+    "type": "table",
+  },
+]
+`;
+
```

Comment:
![image](https://user-images.githubusercontent.com/19568747/140654859-b5d903a3-ae4f-4721-aee7-1198b322369c.png)

```
table:node in table cells
	#hashtag
	[* deco]
	[ ]
	`code`
	https://external.com
	[https://external.com]
	[left https://external.com]
	[https://external.com right]
	[$ x]
	[N35.6812362,E139.7649361]
	#hashTag
	? helpfeel
	$ commandline
	[progfay.icon]
	[https://image.com/image.png]
	[link]
	plain
	> quote
	[[progfay.icon]]
	[[https://image.com/image.png]]
	[[strong]]
```

Reply:
I checked that the snapshot matches to actual behavior (scrapbox.io).